internal/common/build_hook.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type BuildHook struct {
+}
+
+func (h *BuildHook) Levels() []logrus.Level {
+	return []logrus.Level{
+		logrus.DebugLevel,
+		logrus.InfoLevel,
+		logrus.WarnLevel,
+		logrus.ErrorLevel,
+		logrus.FatalLevel,
+		logrus.PanicLevel,
+	}
+}
+
+func (h *BuildHook) Fire(e *logrus.Entry) error {
+	e.Data["build_commit"] = BuildCommit
+	e.Data["build_time"] = BuildTime
```

Comment:
do we need build_time for every log line? It sounds a little excessive :thinking:
Wouldn't commit be enough? I'm just bit worried about overpolluting the logs.